raptor/tests/raptor-speedometer.ini

```diff
@@ -0,0 +1,12 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# raptor speedometer
+
+[raptor-speedometer]
+apps = firefox chrome
```

Comment:
Again not sure best way to have multiple items in one INI key... do you think this is better with no commas?

Reply:
Yeah, I like space delimiting better. It lets you list items like this:
https://searchfox.org/mozilla-central/source/dom/imptests/editing/mochitest.ini#3

Reply:
oh cool thanks